Vsts-Extension/Set-AzureAdApplication/Set-AzureAdApplicationV2/task.json

```diff
@@ -222,6 +231,10 @@
         {
             "name": "ServicePrincipalObjectId",
             "description": "The ObjectId of the Service Principal connected to the Azure AD Application"
+        },
+        {
+            "name": "Secret.$Description",
```

Comment:
I this a dynamic name now because of the $ sign?
So in in example I now get the output variable "Secret.MyDescription" and "Secret.Foo"?
How is this filled? Or is this just a placeholder now? If so, add "Secret.MyDescription" as sample in the text.

Reply:
Working on that, I don't think there is a way to dynamically show all secrets as output variables at design-time. However, since we're using `Write-Host "##vso[task.setvariable variable=Secret.$secret.Description;]$($appKeySecret.Value)"`  they should be available as environment variables.

Reply:
@LockTar I changed it to 'MyDescription' as per your suggestion

Reply:
I agree. this is enough